resources/definitions/hctech_hc300-m5.def.json

```
@@ -0,0 +1,52 @@
+{
+    "version": 1,
```

```suggestion
    "version": 2,
```
This is the version of the file format, not the version how often it was updated